packages/app/src/runner/event-manager.ts

```diff
@@ -537,8 +537,8 @@ export class EventManager {
     Cypress.on('after:screenshot', handleAfterScreenshot)
 
     driverTestEvents.forEach((event) => {
-      Cypress.on(event, (test, cb) => {
-        this.reporterBus.emit(event, test, cb)
+      Cypress.on(event, (test, _runnable) => {
```

Comment:
It honestly might be worth deleting `driverTestEvents` and breaking these out to match their apis: 🤷🏻‍♀️ 

```js
Cypress.on('test:before:run:async', (test, _runnable) => {
   this.reporterBus.emit(event, test)
})

Cypress.on('test:after:run', (test, _runnable) => {
  this.reporterBus.emit(event, test, Cypress.config('isInteractive'))
})
```

Reply:
updated [4e8452e](https://github.com/cypress-io/cypress/pull/25678/commits/4e8452e9054088751d543a0c347b15644e8c3d97)